amqphandler/amqphandler.go

```diff
@@ -805,3 +785,32 @@ func (handler *AmqpHandler) sendMessage(
 
 	return nil
 }
+
+func isMessageImportant(message interface{}) bool {
```

Comment:
What is the benefit having function with switch instead of array?

Reply:
I choose among two options:
using type switch &
array with typenames.
But the latter requires quite redundant syntax to make an array,
like :
reflect.TypeOf(cloudprotocol.DesiredStatus{}).String(), reflect.TypeOf(cloudprotocol.StateAcceptance{}).String(),